src/OrchardCore.Modules/OrchardCore.Users/TwoFactorAuthenticationStartup.cs

```diff
@@ -38,9 +38,9 @@ public override void ConfigureServices(IServiceCollection services)
         services.Configure<MvcOptions>(options =>
         {
             options.Filters.Add<TwoFactorAuthenticationAuthorizationFilter>();
-            options.Filters.Add<LoginMenuFilter>();
         });
 
+        services.AddScoped<IDisplayDriver<UserMenu>, UserMenuDisplayDriver>();
```

Comment:
Looks like this one is also registered in the other Users Startup

Reply:
good catch @jtkech . Fixed!